pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/worker/WorkerConfig.java

```diff
@@ -179,6 +179,11 @@
         category = CATEGORY_FUNCTIONS,
         doc = "The path to the location to locate builtin functions"
     )
+    private Boolean uploadBuiltinSinksSources = true;
```

Comment:
why not `boolean` ?

Reply:
with lombok it only matters if it generates accessor methods like "getBoolean" vs "isBoolean".

Reply:
Ok but I think using 'is' better then 'get' for Java booleans, btw we can keep in this way, no problem